packages/validation-errors/package.json

```diff
@@ -25,8 +25,7 @@
     "@contentful/forma-36-react-components": "^3.40.1",
     "@contentful/forma-36-tokens": "^0.7.1",
     "emotion": "^10.0.17",
-    "lodash": "^4.17.15",
-    "lodash-es": "^4.17.15"
+    "lodash": "4.17.16"
```

Comment:
Could you use "lodash": "^4.17.15" so we don't have a concrete locked version here. With a concrete locked version you often end up with several lodash packages in the final bundle.

Reply:
Oh, yeah, my bad :+1:

Reply:
Or you simply get rid of lodash completely in this package by doing a diff function on your own 😄

Reply:
@anho done. You can enjoy your vacation now :wink: